plugins/modules/s3_bucket_notification.py

```diff
@@ -95,17 +112,37 @@
     state: present
     event_name: on_file_add_or_remove
     bucket_name: test-bucket
-    function_name: arn:aws:lambda:us-east-2:526810320200:function:test-lambda
+    lambda_function_arn: arn:aws:lambda:us-east-2:526810320200:function:test-lambda
     events: ["s3:ObjectCreated:*", "s3:ObjectRemoved:*"]
     prefix: images/
     suffix: .jpg
+
+# Example removing an event notification
+- name: Remove event notification
+  community.aws.s3_bucket_notification:
+    state: absent
+    event_name: on_file_add_or_remove
+    bucket_name: test-bucket
```

Comment:
Can you please add another example with `queue_arn` or  `queue_arn ` or both?

Reply:
Done @alinabuzachis :)